### DISCUSSION TITLE
1 alert rule to more Contact Points?

### DISCUSSION THREAD

**Discussion by nbotulj** [grafana/grafana#61275] [Alerting]
Hello,
when I set one alert rule notification to go to two contact points (both type: email), only one actually receives an email. Is there a possibility to somehow turn on option so both CPs receive  emails?
I know that I can add more emails to one CP, and that I can assign different types to one CP.

In  example below (pic) only CP DBZ will recieve email alert (Edo won't).
![Matching-labels](https://user-images.githubusercontent.com/122441359/211796830-402198c8-60f3-4538-bcca-99273013d172.png)

**Comment by vbichov** [comment]
just wondering if you have `Continue matching sibling nodes` checked
<img width="758" alt="Screen Shot 2023-01-12 at 13 29 47" src="https://user-images.githubusercontent.com/4352719/212055882-f5ad6c04-76fb-4147-b229-304f35626110.png">

**Reply by nbotulj** [reply]
That was the case @vbichov. Thank you :-)
Grafana was provisioned and notification policy couldn't be edited so i forgot about that option at all. 

I added it to deployment file like this:
![image](https://user-images.githubusercontent.com/122441359/212479428-c2363274-cac0-41a8-a45f-16b5087f73b8.png)

and now it works like charm :-)

**Comment by gelicia** [comment]
Closing, as the question was answered.
